lib/trino-filesystem-s3/src/main/java/io/trino/filesystem/s3/S3FileSystem.java

```diff
@@ -398,10 +397,8 @@ private static void validateS3Location(Location location)
     // TODO (https://github.com/trinodb/trino/issues/24955):
     // remove me once all of the S3-compatible storage support strong integrity checks
     @SuppressWarnings("deprecation")
-    static AwsRequestOverrideConfiguration disableStrongIntegrityChecksums()
+    static AwsRequestOverrideConfiguration.Builder disableStrongIntegrityChecksums(AwsRequestOverrideConfiguration.Builder builder)
```

Comment:
We added `LegacyMd5Plugin` in #25711. Why do we still need this method?
cc: @wendigo

Reply:
https://github.com/trinodb/trino/pull/25791 this should help I think